spec/models/notice_submission_finalizer_spec.rb

```diff
@@ -32,21 +32,49 @@
     expect(notice.submission_id).to eq notice.id
   end
 
+  it 'increases the count of domains involved in the notice' do
+    notice = build(:dmca)
+    notice.save
+    
+    counter_map = {}
+    distinct_domains = infringing_urls_attributes.map { 
+      |x| x[:url].start_with?("http") ? Addressable::URI.parse(x[:url]).domain : PublicSuffix.domain(x[:url]) 
+    }.uniq
+    distinct_domains.each do |domain|
+      counter_map[domain] = return_count(DomainCount.find_by_domain_name(domain))
+    end
+
+    NoticeSubmissionFinalizer.new(
+      notice, works_attributes: works_attributes
+    ).finalize
+
+    distinct_domains.each do |domain|
+      expect(return_count(DomainCount.find_by_domain_name(domain)) - counter_map[domain]).to eq(1)
+    end
+  end
+
+  it 'handles large url attributes list efficiently' do
```

Comment:
What is the goal of this test? It does not assert anything, so I'm not sure what behavior it's verifying -- it might just be slowing us down.